### PR TITLE
Geodatabase transactions

### PR DESCRIPTION
**Pull request by ThadT**
Samples for all platforms that illustrate start transaction, rollback, commit for managing edits to a local geodatabase.

### CODE REVIEW

**Comment by ThadT** [comment]
Hey @macca400, I thought you might have time to review while we're eating turkey ;)
No hurry if you can't get to this right away. Thanks!

**Comment by ThadT** [comment]
Hey @macca400
I know you're busy ... will you be able to review this one before the release?

**Comment by macca400** [comment]
@ThadT I'll look at it on Monday

File: src/iOS/Xamarin.iOS/Samples/Data/GeodatabaseTransactions/GeodatabaseTransactions.cs [hunk 1]
```diff
@@ -0,0 +1,562 @@
...
+                            // report failure ...
+                                _messageTextBlock.Text = job.Error.Message;
+                                _progressBar.Hidden = true;
+                        }
+                        else
+                        {
+                            // report that the job is in progress ...
+                            _messageTextBlock.Text = "Sync in progress ...";
+                        }
+
```
Comment (macca400) [inline src/iOS/Xamarin.iOS/Samples/Data/GeodatabaseTransactions/GeodatabaseTransactions.cs]: whitespace

**Pull request merged**